cmd/headscale/cli/utils.go

```diff
@@ -221,10 +221,20 @@ func getHeadscaleConfig() headscale.Config {
 	dnsConfig, baseDomain := GetDNSConfig()
 	derpConfig := GetDERPConfig()
 
+	configuredPrefixes := viper.GetStringSlice("ip_prefixes")
```

Comment:
This needs to be documented, and added in the `config-example.yml`

Reply:
Example config updated.